src/app/TableMsg.elm

```diff
@@ -0,0 +1,13 @@
+module TableMsg exposing (Msg(..))
```

Comment:
Perhaps the main update `Msg` type shouldn't be in `Types` then?

I'm not sure how we can embed the `Table` page within the main app without also embedding its `Msg` type within the main `Msg` type.

WDYT? Any ideas?